Dockerfile

```diff
@@ -1,24 +1,17 @@
-FROM node:8.12.0-slim
-
-WORKDIR /usr/src/app
-
-COPY package.json yarn.lock ./
-RUN curl -o- -L https://yarnpkg.com/install.sh | bash -s \
-    && export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH" \
+# ---- Base Image ----
+FROM hmcts.azurecr.io/hmcts/base/node/stretch-slim-lts-8 as base
+RUN export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH" \
     && yarn install --production \
     && yarn cache clean
+COPY package.json yarn.lock ./
 
+# ---- Build Image ----
+FROM base as build
+COPY . .
+RUN yarn install
 
-COPY app.js server.js ./
-COPY app ./app
-COPY config ./config
-
-ENV PORT 3460
-
-HEALTHCHECK --interval=10s \
```

Comment:
Is the `HEALTHCHECK` part of the base image?

Reply:
No, however the AKS config should come with its own healthcheck (liveness probes) so this instruction might be redoundant.

Afaik we do not intend adding any healthcheck at the container level.